core/jvm/src/test/scala/scalaz/zio/QueueSpec.scala

```diff
@@ -1,13 +1,14 @@
 package scalaz.zio
 
 import scala.collection.immutable.Range
-import scala.concurrent.duration._
 import org.specs2.concurrent.ExecutionEnv
 import org.specs2.specification.AroundTimeout
 
 class QueueSpec(implicit ee: ExecutionEnv) extends AbstractRTSSpec with AroundTimeout {
 
-  def is =
+  def is = {
+    import scala.concurrent.duration._
```

Comment:
Do we need this import?

Reply:
`AroundTimeout.upTo` take `concurrent.duration.Duration` as an argument. Not sure what's the best solution would be to avoid mixing it with scalaz Duration

Reply:
No one should be using `upTo` anymore in any of the specs that extend `AbstractRTSSpec` (it's built into the parent).